scipy/special/special/cephes/polevl.h

```diff
@@ -115,6 +116,7 @@ namespace cephes {
 
     /* Evaluate a rational function. See [1]. */
 
+    /* The function ratevl is only used once in cephes/lanczos.h. */
```

Comment:
I think it's fine to keep this comment here, but just want to point out that this may change in the future, especially as we work on unifying the `cephes`, `amos`, `specfun`, `cdflib`, `faddeeva`, etc. codebases.

Reply:
Sure. I just checked the code for $erf(x)$ and $erfc(x)$ in `special/Faddeeva.cc`, which I think have better performance than the ones in `cephes/ndtr.h`, so is it possible to improve the performance of the functions in `cephes/ndtr.h` in the future, i.e., $erf(x)$, $erfc(x)$, and $ndtr(x)$?.

Reply:
We actually found that `cephes` has better accuracy for real valued `ndtr`, `erf`, and `erfc`. See https://github.com/scipy/scipy/pull/19492#issuecomment-1802330899.

I just meant that it's not unlikely that `ratevl` will end up being used again in another function, and I want to make sure to keep in mind that if so, this comment will need to be updated.